client/src/index.css

```diff
@@ -329,10 +325,21 @@ li#search-nav .nav > li a {
 	padding-right: 4px;
 }
 
-@media (min-width: 768px) {
-	.nav-fixed {
-		position: fixed;
-	}
+.nav-fixed {
+	flex:0 0 auto;
+	padding:8px;
+	overflow-y:scroll;
+}
+
+.nav-overlay {
+	padding:8px;
+	border: 1px solid #e0e0e0;
+    height: 100vh;
```

Comment:
Indentation is off.

Reply:
337 had 4 whitespaces, while the other lines are tabs. in my editor it looks flush. I am changing it to a tab to be consistent. I assume the tools we are planning to use will enforce it one way or the other